tests/test_layout.py

```diff
@@ -10,8 +10,6 @@
 class TestLayout:
 
     @pytest.mark.nondestructive
-    @pytest.mark.xfail("'mozilla.com' in config.getvalue('base_url')",
-                       reason="https://bugzilla.mozilla.org/show_bug.cgi?id=1201622")
```

Comment:
It was a change that was made to stage a long time ago, the change propagated out to prod. There were some odd disagreements on the team and so I left the test as it was for awhile. I've ping'd on the bug for a final resolution.